build.sh

```diff
@@ -53,10 +53,19 @@ sed -i -e "s/BUILD_DATE/${DATE}/g" maestroutils/status.go
 # highlight errors: https://serverfault.com/questions/59262/bash-print-stderr-in-red-color
 # color()(set -o pipefail;"$@" 2>&1>&3|sed $'s,.*,\e[31m&\e[m,'>&2)3>&1
 
+#Developers may have a GOBIN defined. If not define it to current dir.
+if [ -z "$GOBIN" ]
+then
+  GOBIN=./bin
+fi
+
 if [[ "${1:-}" != "preprocess_only" ]]; then
-	if [[ -n "${TIGHT:-}" ]]; then
-	    go build "${GOTAGS}" -ldflags="-s -w" "$@" github.com/armPelionEdge/maestro/maestro 
-	else
-	    go build "${GOTAGS}" "$@" github.com/armPelionEdge/maestro/maestro 
-	fi
+  mkdir -p $GOBIN
```

Comment:
```suggestion
  mkdir -p "${GOBIN}"
```

Add brackets to fail if the variable is undefined, and double quotes to make sure variable is treated only as a string (ie, what would/could happen if the variable has a space).